.github/workflows/agw-coverage.yml

```diff
@@ -0,0 +1,242 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: agw-coverage
```

Comment:
@voisey Hm, I guess `AGW Generate Coverage` would be it according to your scheme, unless we want to allow `AGW Coverage` and add a new verb?

Reply:
I think `AGW Generate Coverage` sounds good. I guess if we wanted to add a new verb, it would have to be `AGW Cover` or something, which sounds a little strange to me